server/storage.go

```diff
@@ -76,68 +72,24 @@ func (s *MemoryStorage) Store() storage.Store {
 	return s.store
 }
 
-type BadgerStorage struct {
-	logger         *logrus.Logger
-	store          *badger.Store
-	ticker         *time.Ticker
-	done           chan bool
-	gcInterval     time.Duration
-	gcDiscardRatio float64
+type SqliteStorage struct {
```

Comment:
Noice.... I wonder if it would make sense to use an SQL adapter library so we could switch between different SQL DBMS? Not sure if worth it, just an idea maybe.

Reply:
I was thinking about this when implementing, maybe on another PR, we need to make a smart config.